hw/ip/lc_ctrl/doc/_index.md

```diff
@@ -195,41 +200,220 @@ This ensures that during specific states (RAW, TEST_LOCKED, SCRAP, INVALID) it i
+
+### ID State of the Device
+
+If the SECRET2_DIGEST is zero, the device is considered to have "blank" ID state, in which case the CREATOR_ROOT_KEY_* (in OTP) and CREATOR_DIV_KEY (in FLASH) can be written by software.
+All consumers of these keys are supplied with an invalid value.
+
+If the SECRET2_DIGEST has a nonzero value, the device is considered "creator personalized", and the CREATOR_ROOT_KEY and CREATOR_DIV_KEY are no longer accessible to software.
+Actual values are supplied to the consumers.
+If SECRET2_DIGEST has a nonzero value, the CREATOR_SEED_SW_RW_EN signal will be disabled in PROD, PROD_END and RMA states.
+
+### Secret Collateral
+
+Among the OTP life cycle collateral, the following are considered secrets (note there may be other secrets unrelated to life cycle, please see [OTP controller specification]({{< relref "hw/ip/otp_ctrl/doc/_index.md#partition-listing-and-description">}}) for more details):
+
+- *_TOKEN
+- CREATOR_ROOT_KEY*
+
+Specifically this means after OTP sensing, the above entries are unwrapped to obtain the real value.
+Similarly, during programming, they are wrapped before beginning to be written to OTP.
+
+The function used for this wrapping is the lightweight PRESENT-cipher.
+The wrapping is a one time event during controlled manufacturing, and unwrapping also cannot be supplied with arbitrary ciphertexts.
+Thus the system cannot be abused to generate a large number of traces for informational leakage, and thus a fully hardened cipher (such as AES) is not required.
+
+Note also, a global key is used here because there is no other non-volatile location to store a secret key.
+If PUFs were available (either in memory form or fused form), it could become an appealing alternative to hold a device unique fuse key.
+
+See the [OTP controller]({{< relref "hw/ip/otp_ctrl/doc/_index.md#secret-vs-non-secret-partitions">}}) for more details.
+
+### OTP Accessibility Summary and Impact of Life Cycle Signals
+
+A subset of secret collateral is further access-controlled by the life cycle CREATOR_SEED_SW_RW_EN signal.
+These are
+
+- RMA_UNLOCK_TOKEN
+- CREATOR_ROOT_KEY
+
+The table below summarizes the software accessibility of all life cycle collateral.
+
+{{< snippet "hw/ip/lc_ctrl/doc/lc_ctrl_otp_accessibility.md" >}}
+
+Note that CREATOR_SEED_SW_RW_EN is set to OFF if SECRET2_DIGEST has a nonzero value in PROD, PROD_END and DEV states.
+SEED_HW_RD_EN only becomes active if SECRET2_DIGEST has a nonzero value in DEV, PROD, PROD_END and RMA states.
+
+## Flash Collateral
+
+The flash contains both memory mapped and non-memory mapped partitions.
+As it pertains to life cycle, the flash contains three sets of important collateral.
+They are enumerated in the table below.
+Just as with OTP, the consumer and usage of each is also described.
+
+{{< snippet "hw/ip/lc_ctrl/doc/lc_ctrl_flash_collateral.md" >}}
```

Comment:
this is my bad for not updating. But at the moment `ownership blob` does not exist.
So you can take it out of the table.  (fine to be different PR)

Reply:
ok let me remove that (I can do it here since there are a few other things that need fixing)